### ISSUE TITLE
Problem with linked products label.

### ISSUE THREAD

**Issue by trallawan** [qTranslate-Team/woocommerce-qtranslate-x#43]
I have an issue with translating linked products in the grouped product types. At the moment it has to show the title of the linked products in the product description, it shows the label as the full multilingual definition instead (showing the names of the products in the two languages with the [:es]...[:en] prefixes)

Clearly the plugin is not reading the correct label-for="product:***" in its appropiate form.

here is a link to one of the products in case you need to see it. See how both linked products show the label wrong:
https://www.naturtrap.com/tienda/soporte-mesa-de-cultivo/

I have no clue on how to fix this. I appreciate any help you can provide
Thank you.

**Comment by trallawan** [comment]
SOLVED
I didn't have much expectatives to have a quick answer... but luckily I managed to figure out how to solve this. And I'll post how I did it, just do as follows:
-Go to templates, single product, addtcart folder and edit grouped.php
-just get the line with 
**$grouped_product->get_name() . '</a>' : $grouped_product->get_name(); ?>**

right at the end of the line... and place this instead:
**__($grouped_product->get_name()) . '</a>' : __($grouped_product->get_name()); ?>**

Basically allow the translation function to retrieve the string corresponding to the product label.

Thank me later...
